src/WikiService/Controllers/ArticleController.cs

```diff
@@ -18,5 +19,13 @@ public class ArticleController: ControllerBase
     {
       return await command.ExecuteAsync(request);
     }
+
+    [HttpGet("get")]
+    public async Task<OperationResultResponse<ArticleResponse>> GetAsync(
+      [FromServices] IGetArticleCommand command,
+      [FromQuery] GetArticleRequest request)
```

Comment:
you can add ArticleId here instead of request, if request won't expand in the future